VMS/Pages/ActivitySearchPage/Pagination.razor

```diff
@@ -0,0 +1,210 @@
+                        <div class="col-9 act__content">
+                            <h3 class="lmtext-3">
+                                Tuyển tình nguyện viên tham gia công tác phòng, chống dịch bệnh Covid-19 tại địa
+                                phương
+                            </h3>
+                            <h5><i class="material-icons">public</i>Tên đơn vị tổ chức</h5>
+                            <p class="lmtext-2">
+                                Phường Phú Hữu thành phố Thủ Đức trân trọng kính mời y bác sĩ, cán bộ y tế nghỉ hưu
+                                có sức khoẻ, tinh thần hăng hái.
+                            </p>
+                            <span>Ngày đăng: 20/07/202</span>
+                            <div class="act__signUp">
+                                <i class="material-icons act__notLike">favorite_border</i>
+                                <i class="material-icons act__like">favorite</i>
+                                <button @onclick="ShowModal" class="btn">Đăng kí</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <a class="goToDivAct d-none " href="#act">go to div act</a>
+            </div>
+            <div class="featuredAct col-3 ">
+                <div class="featuredAct__content">
+                    <div class="featuredAct__card ">
+                        <img class="w-100" src="img/act1.png" alt="">
+                        <div class="featuredAct__title">
+                            <h3 class="lmtext-5">Tuyển tình nguyện viên tham gia công tác phòng, chống dịch bệnh Covid-19 tại địa phương</h3>
+                            <div class="featuredAct__overplay"></div>
+                        </div>
+                    </div>
+                    <div class="featuredAct__card">
+                        <img class="w-100" src="img/act1.png" alt="">
+                        <div class="featuredAct__title">
+                            <h3>Tuyển tình nguyện viên tham gia công tác phòng, chống dịch bệnh Covid-19 tại địa phương</h3>
+                            <div class="featuredAct__overplay"></div>
+                        </div>
+
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </div>
+
+</section>
+<section class="pagination">
+    <ul>      
+        <li class="btn-prev"><span class="material-icons">keyboard_arrow_left</span></li>
+        <li class="numb">1</li>
+        <li class="dots">...</li>
+        <li class="numb">4</li>
+        <li class="numb">5</li>
+        <li class="numb">6</li>
+        <li class="dots">...</li>
+        <li class="numb">10</li>
+        <li class="btn-prev"><span class="material-icons">keyboard_arrow_right</span></li>
+    </ul>
+</section>
+
+
+
+
+@code {
+
+
+    protected override async Task OnAfterRenderAsync(bool firstRender)
+    {
+        await JSRuntinme.InvokeVoidAsync("vms.ShowColor");
```

Comment:
Please don't use JS to toggle the Like button.
@hoanguyennph has implemented it successfully, you can check it out for details: https://github.com/UEH-Squad/VMS/blob/3b1e130d03245c4071d4904b263cae597d4e541b/VMS/Pages/ActivityInfoPage/InfoPage.razor#L61